README.md

```diff
@@ -87,6 +92,32 @@ make stop
 make clean
 ```
 
+## Deploying
+
+All deployments are run by the [GDS Concourse][gds-concourse].
+
+The process amounts to:
+
+1. Test + Lint
+2. Build a deployment image
+3. Push deployment image to the GovWifi ECR
+4. Run migrations using the deployment image
+5. restart the ECS services using the new deployment image
+
+### Staging
+
+Deployments to Staging are run automatically whenever a change to the `master` branch is made.
```

Comment:
If I wanted to push a commit straight through to production, do I start by going to the Lint and Test task?

Reply:
nope. You can click it, and it will wait for staging + tests to finish